mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.thefuntasty.mvvm.test
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import io.github.plastix.rxschedulerrule.RxSchedulerRule
+import org.junit.Rule
+
+/**
+ * Base class for ViewModel testing.
+ */
+abstract class ViewModelTest {
+
+    /*
+    class SampleViewModelTest : ViewModelTest() {
+
+        val mockViewState: SampleViewState = mockk()
+
+        lateinit var viewModel: SampleViewModel
+
+        @Before
+        fun setUp() {
+            viewModel = spyk(SampleViewModel(mockViewState), recordPrivateCalls = true)
+        }
+
+        @Test
+        fun `when ... then ...`() {
+            // GIVEN
+            ...
+
+            // WHEN
+            ...
+
+            // THEN
+            ...
+        }
+    }
+    */
+
+    /**
+     * Swap RxJava schedulers with the ones that are executed immediately.
+     * It allows to work with the RxJava.
+     */
+    @get:Rule var rxJavaRule = RxSchedulerRule()
+
+    /**
+     * Swap background android executor with the one that executes task synchronously.
+     * It allows to work with the live data.
+     */
+    @get:Rule val instantTaskExecutorRule = InstantTaskExecutorRule()
+}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/ef3136579489660a161140b43ae3c9f625f10e7b/mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt#L49"></span>The file ViewModelTest.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>